### ISSUE TITLE
Splits rest api returns a JSON Array

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#2491]
It would probably be more future-friendly and standard friendly to return a json object (although really i am not sure about that!?)

**Comment by fmassot** [comment]
Indeed, I'm in favor of a JSON object. It is practical for pagination, we could, for example, add the total num hits, the offset, and the limit to ease client usage.
